api/tests/unit/infrastructure/serializers/csv/higher-education-registration-parser_test.js

```diff
@@ -74,4 +74,94 @@ describe('Unit | Infrastructure | HigherEducationRegistrationParser', () => {
       });
     });
   });
+
+  context('When file does not match requirements', () => {
+    const organizationId = 123;
+    it('should throw an error if the file is not csv', async () => {
+      const input = `Premier prénom\\Deuxième prénom2\\Troisième prénom\\Nom de famille\\Nom d’usage\\Date de naissance (jj/mm/aaaa)\\Email\\Numéro étudiant\\Composante\\Équipe pédagogique\\Groupe\\Diplôme\\Régime
```

Comment:
```suggestion
      const input = `Premier prénom\\Deuxième prénom\\Troisième prénom\\Nom de famille\\Nom d’usage\\Date de naissance (jj/mm/aaaa)\\Email\\Numéro étudiant\\Composante\\Équipe pédagogique\\Groupe\\Diplôme\\Régime
```

Reply:
Non c'est fait exprès pour le test. On vérifie les colonnes inconnues ou qui ont une faute de frappe 👍

Reply:
pas ce test là, si j'en crois `should throw an error if the file is not csv`

Reply:
Ah oui mince, j'ai répondu un peu vite !